lib/components/map/connected-park-and-ride-overlay.js

```diff
@@ -0,0 +1,40 @@
+import ParkAndRideOverlay from '@opentripplanner/park-and-ride-overlay'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { setLocation } from '../../actions/map'
+import { parkAndRideQuery } from '../../actions/api'
+
+class ConnectedParkAndRideOverlay extends Component {
+  componentDidMount () {
+    const params = {}
+    if (this.props.maxTransitDistance) {
+      params['maxTransitDistance'] = this.props.maxTransitDistance
```

Comment:
I'm confused as to why you're referencing the key in this way. Why not use dot notation?